pritunl/sso/azure.py

```diff
@@ -203,6 +203,8 @@ def _verify_azure_2(user_name):
             display_name = group_data.get('displayName')
             if not display_name:
                 continue
+            if settings.app.sso_azure_allow_groups and display_name.replace(" ","_").replace("'","") not in settings.app.sso_azure_allow_groups:
```

Comment:
The reason for replacing spaces and apostrophe's in the group display names are because its not possible to add groups with space names on the server settings (UI separates them) and the apostrophe's are also removed when adding/saving them.